tests/testthat/test-get_baseline_predictions.R

```diff
@@ -0,0 +1,322 @@
+  target_ts |>
+    dplyr::mutate(location = "ak2") |>
+    dplyr::bind_rows(target_ts) |>
+    get_baseline_predictions(transformation = "none",
+                             symmetrize = TRUE,
+                             window_size = 3,
+                             effective_horizons = 1:4,
+                             origin = "obs",
+                             quantile_levels = c(.1, .5, .9),
+                             n_samples = NULL) |>
+    expect_error(
+      regexp = " but only one may be provided.", fixed = TRUE
+    )
+})
+
+test_that("invalid origin value throws an error", {
+  get_baseline_predictions(target_ts,
+                           transformation = "none",
+                           symmetrize = TRUE,
+                           window_size = 3,
+                           effective_horizons = 1:4,
+                           origin = "x",
+                           quantile_levels = c(.1, .5, .9),
+                           n_samples = NULL) |>
+    expect_error(
+      regexp = "`origin` must be only one of ", fixed = TRUE
+    )
+})
+
+test_that("invalid quantile_levels value throws an error", {
+  get_baseline_predictions(target_ts,
+                           transformation = "none",
+                           symmetrize = TRUE,
+                           window_size = 3,
+                           effective_horizons = 1:4,
+                           origin = "obs",
+                           quantile_levels = c(1, 5, 9),
+                           n_samples = NULL) |>
+    expect_error(
+      regexp = "`quantile_levels` must only contain values between 0 and 1.", fixed = TRUE
+    )
+})
+
+test_that("not requesting any forecasts throws an error", {
+  get_baseline_predictions(target_ts,
+                           transformation = "none",
+                           symmetrize = TRUE,
+                           window_size = 3,
+                           effective_horizons = 1:4,
+                           origin = "obs",
+                           quantile_levels = NULL,
+                           n_samples = NULL) |>
+    expect_error(
+      regexp = "No forecasts requested: both `quantile_levels` and `n_samples` are NULL", fixed = TRUE
+    )
+})
+
+# correct transformation/symmetrize/window_size applied
+
+test_that("only non-negative forecast values are returned", {
+  sample_predictions <- target_ts |>
+    get_baseline_predictions(transformation = "none",
+                             symmetrize = TRUE,
+                             window_size = 3,
+                             effective_horizons = 1:4,
+                             origin = "obs",
+                             quantile_levels = NULL,
+                             n_samples = 10000,
+                             round_predictions = FALSE,
+                             seed = 1234) |>
+    tidyr::unnest(cols = c("forecasts"))
+
+  min_value <- min(sample_predictions$value)
+  if (min_value != 0) {
```

Comment:
could we just use a single `expect_gte` here?

Reply:
didn't realize there was a combined expectation — it's been fixed